File: 프로그래머스/2/64065. 튜플/튜플.py
def solution(s): #문자열
    answer = []
    
    s = s[2:-2].split("},{")
    s = sorted(s, key = len)
    
    for x in s: #튜플을 하나씩 꺼내서
        for num in x.split(','): #해당 튜플을 순회
            if not int(num) in answer:
                answer.append(int(num))
            
    return answer

File: 프로그래머스/2/64065. 튜플/튜플.js
function solution(s) {
    //집합 기호 제거 후, 길이로 정렬
    const tuples = s.slice(2, -2).split("},{").sort((a,b) => a.length-b.length)
    
    const result = tuples.reduce((acc, elements) => {
        let splitedTuple = elements.split(',')
        let t = splitedTuple.find(e => !acc.includes(e))
        acc.push(t) 
        return acc
    }, [])
    
    console.log(result)
    return result.map(Number);
}